go.mod

```diff
@@ -1,24 +1,77 @@
 module github.com/openshift/elasticsearch-operator
 
-go 1.16
+go 1.17
 
-// Pinned to kubernetes-1.21.3 and openshift release-4.8
 require (
-	github.com/ViaQ/logerr v1.0.10
+	github.com/ViaQ/logerr v1.1.0
```

Comment:
To use the `log.DefaultLogger()` w/o wasting resource, I suggest you bump this to 1.1.1 released today.